build.xml

```diff
@@ -155,7 +155,7 @@
 	<target name="unit" depends="unit-build" unless="${test.skip}"
 	        description="Runs the unit tests">
 		<mkdir dir="${test.run}"/>
-		<junit fork="yes" dir="${test.run}">
+		<junit fork="yes" dir="${test.run}" haltonfailure="true">
```

Comment:
From the [docs](https://ant.apache.org/manual/Tasks/junit.html) this defaults to `off`. Does `"true"` work or should it be `"yes"`?